Make-a-pull-request.md

```diff
@@ -0,0 +1,168 @@
+
+     * Ensure that your code is fully covered by automated tests. (For more info, see the wiki pages on writing tests in the sidebar menu.)
+
+     **We don't allow force-pushing at Oppia, so once you push your commits, you can't change them.** You can still add new commits though.
+
+   * When you're ready to push, run:
+
+     ```console
+     git push origin {{YOUR BRANCH NAME}}
+     ```
+
+     For more convenience, you can also run:
+
+     ```console
+     git push -u origin {{YOUR BRANCH NAME}}
+     ```
+
+     Then in the future, you can just run `git push`.
+
+     **Make sure to do this from the command line** (and not GitHub's Desktop client), since this also runs some important presubmit checks before your code gets uploaded to GitHub. If any of these checks fail, read the failure messages and fix the issues by making a new commit (see step 3), then **repeat the previous instructions** to retry the push. **Do not bypass these presubmit checks!** The checks get run on your PR too, so if you bypass failures on your local machine, you'll just have to fix them later when they fail on the PR.
+
+4. **When your feature is ready to merge, create a pull request.**
+
+   * Go to your fork on GitHub, select your branch from the dropdown menu, and click "pull request". Ensure that the base repository is `oppia/oppia` and that the base branch is `develop`. The head repository should be your fork, and the head branch should be your branch. If you don't see the repository, click the link to `compare across forks`.
+
+     On this page you can also see your commits and your changes.  Read these changes carefully to make sure that the changes are correct (e.g., no missing newlines at the ends of files; no files left out by mistake). This is a good way to catch obvious errors that would otherwise lead to delays in the review process.
+
+     ![The GitHub PR creation screen](images/makingAPR.png)
+
+   * Click "Create pull request".
+
+   * Following the guidance in the PR checklist, add a descriptive title explaining the purpose of the PR (e.g. "Fix issue #bugnum: add a warning when the user leaves a page in the middle of an exploration.").
+
+   * Fill out the rest of the PR checklist.
+
+   * **Test your code to confirm that the changes you have made are working properly** Test all the places where the changes are made, including buttons, tabs, lists etc. Make a screen recording of these changes and add it to the description.
+
+   * Click "Create pull request".
+
+   * If you have already completed 2 pull requests and been added as a collaborator to the project, you should also add a changelog label.  If you are a new contributor, you don't have permission to do this.  Don't worry! Oppiabot will automatically ask someone to do it for you.
+
+   * Oppiabot will check that you filled out the PR description correctly. If you didn't, it will leave a comment explaining what you need to do to fix it.
+
```

Comment:
I think we should let them know that Oppiabot will give them instructions (as we do here), but also tell them what those steps should be in case they have trouble understanding Oppiabot's message. Personally, I find instructions that say "just run the program and follow the prompts" to be frustrating and unhelpful because I'm often looking up the instructions because I was confused by the prompts

Reply:
DO you think we should have a separate wiki page explaining how to resolve OppiaBot's comment? (Which can be managed by OppiaBot team and over here we can ask them to follow OpiaBot's comment and link to the OppiaBot's related wiki page?)

Reply:
Yes, I think a separate page for oppiabot comments would be good. I'll run that idea by the oppiabot team

Reply:
@vojtechjelinek is discussing this with the Oppiabot team. How about we merge this PR and then add a link to the Oppiabot page once it's ready?

Reply:
SGTM!

Reply:
done